light-prover/prover/extractor.go

```diff
@@ -13,20 +13,39 @@ func ExtractLean(treeDepth uint32, numberOfCompressedAccounts uint32) (string, e
 	roots := make([]frontend.Variable, numberOfCompressedAccounts)
 	leaves := make([]frontend.Variable, numberOfCompressedAccounts)
 	inPathIndices := make([]frontend.Variable, numberOfCompressedAccounts)
```

Comment:
shouldn't we have separate `inPathIndices`for `inclusion` and `non-inclusion`?
(right now it seems like the same `inPathIndices` are reused for both `inclusion` (line 29)  and `non-inclusion` (line 41).)